MonoGame.Framework.Content.Pipeline/Audio/DefaultAudioProfile.cs

```diff
@@ -278,14 +278,14 @@ public static ConversionQuality ConvertToFormat(AudioContent content, Conversion
             try
             {
                 string ffmpegCodecName, ffmpegMuxerName;
-                int format;
+                //int format;
                 switch (formatType)
                 {
                     case ConversionFormat.Adpcm:
                         // ADPCM Microsoft 
                         ffmpegCodecName = "adpcm_ms";
                         ffmpegMuxerName = "wav";
-                        format = 0x0002; /* WAVE_FORMAT_ADPCM */
+                        //format = 0x0002; /* WAVE_FORMAT_ADPCM */
```

Comment:
I thought these values might be useful in future, so I left them commented for now.